pkg/errorhandling/errorhandling.go

```diff
@@ -1,6 +1,7 @@
 package errorhandling
 
 import (
+	errors2 "errors"
```

Comment:
Superfluous import - the existing `github.com/pkg/errors` package will work identically.